docs/schema_registry_tutorial.rst

```diff
@@ -0,0 +1,505 @@
+.. _schema_registry_tutorial:
```

Comment:
global: revise content to avoid using `we` - in favor of the second person `you`. (see [Style Guide](https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/161743785/Style+Guide+for+Confluent+Documentation#StyleGuideforConfluentDocumentation-VoiceandTone))